src/client/utils/event-obj-wrapper.js

```diff
@@ -0,0 +1,25 @@
+import fnBind from './fn-bind';
+import nativeMethods from '../sandbox/native-methods';
+
```

Comment:
Add comment why we do this:
`We cannot redefine 'origin' and 'data' properties using Object.defineProperty method. It does not work in IE and Safari. This is why, we clone event properties and replace some of them.`